Dockerfile

```diff
@@ -3,9 +3,8 @@ COPY . /workspace
 WORKDIR /workspace
 ENV CGO_ENABLED=0
 RUN go build -o /workspace/go-pmtiles
-FROM scratch
+FROM gcr.io/distroless/static
 COPY --from=builder /workspace/go-pmtiles /go-pmtiles
 COPY --from=builder etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
```

Comment:
Is it possible we don't need this anymore with `distroless` too?

Reply:
You're probably right, but I don't know how does this line affects the program. I also don't know how to test that nothing is broken after removing it from `Dockerfile`.

I've commented-out this line in a `scratch`-based `Dockerfile`:
```
FROM golang:1.21.8-alpine3.19 AS builder
COPY . /workspace
WORKDIR /workspace
ENV CGO_ENABLED=0
RUN go build -o /workspace/go-pmtiles
FROM scratch
COPY --from=builder /workspace/go-pmtiles /go-pmtiles
# COPY --from=builder etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
ADD root_tmp.tgz /
EXPOSE 8080
ENTRYPOINT ["/go-pmtiles"]
```

and that did not seem to make any difference for `go-pmtiles convert` and `go-pmtiles serve`.

Reply:
In any case, there are certificates in the base image, so no need to add them.

Reply:
The certs are needed for making `pmtiles serve`, `pmtiles show` connections to HTTPS endpoints